server/app.js

```diff
@@ -0,0 +1,160 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var Sequelize = require('sequelize');
+var sequelize = new Sequelize('wonderwander', 'root', null, {
+  dialect: 'mysql',
+  host: 'localhost',
+  port: 3000,
+  allowNull: false // Adds NOT NULL to all values by default
+});
+
+// Router
+// var router = require('./routes.js');
+
+var app = express();
+module.exports.app = app;
+
+
+app.set('port', 3000);
+
+// Clarification on routing -- what does this do?
+// app.use('./', router);
+
+// Serve the client files
+app.use(express.static('public'));
+
+// Define Tables
+var User = sequelize.define('User', {
+  id: {
+    type: Sequelize.INTEGER,
+    unique: true,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+  name: { 
+    type: Sequelize.STRING(25)
+  }
+});
+
+var Itinerary = sequelize.define('Itinerary', {
+  id: {
+    type: Sequelize.INTEGER,
+    unique: true,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  location: Sequelize.STRING(100),
+  numDays: Sequelize.INTEGER,
+  startDate: {
+    type: Sequelize.DATEONLY,
+    allowNull: true
+  },
+  endDate: {
+    type: Sequelize.DATEONLY,
+    allowNull: true
+  },
+  overview: {
+    type: Sequelize.TEXT('long'),
+    allowNull: true
+  }
+});
+
+var Event = sequelize.define('Event', {
+  id: {
+    type: Sequelize.INTEGER,
+    unique: true,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  day: Sequelize.INTEGER,
+  location: Sequelize.STRING(100)
+});
+
+User.hasMany(Itinerary);
+
+Itinerary.belongsTo(User);
+
+Itinerary.hasMany(Event);
+
+Event.belongsTo(Itinerary);
+
+
+
+
+// Routes
+
+// GET - all users
+app.get('/user', function(req, res) {
```

Comment:
@lloversky Correct, we will want to just grab a user, but for the time being we are grabbing all users to check if our request works. Good catch! :)